openlibrary/catalog/importer/db_read.py

```diff
@@ -3,60 +3,68 @@
 import json
 from time import sleep
 
+import requests
+
 from six.moves.urllib.parse import urlencode
-from six.moves.urllib.request import urlopen
 
 
 staging = False
 
 db = web.database(dbn='postgres', db='marc_index', host='ol-db')
 db.printing = False
 
-def find_author(name): # unused
-    iter = web.query('select key from thing, author_str where thing_id = id and key_id = 1 and value = $name', {'name': name })
+
+def find_author(name):  # unused
```

Comment:
Please mark this function with [`@deprecated`](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/marc/fast_parse.py#L16) to mark it for removal in the future.

Reply:
Done.  I tried to test this and web does not have a query attribute, so find_author does not work.